qiskit/utils/quantum_instance.py

```diff
@@ -189,8 +198,9 @@ def __init__(
                         Coupling map (perhaps custom) to target in mapping
             initial_layout (Optional[Union['Layout', Dict, List]]):
                         Initial layout of qubits in mapping
-            pass_manager (Optional['PassManager']):
-                        Pass manager to handle how to compile the circuits
+            pass_manager (Optional['PassManager']): Pass manager to handle how to compile the circuits
+            bound_pass_manager (Optional['PassManager']): Pass manager to apply on bound circuits
+                only.
```

Comment:
Do people generally know what a 'bound circuit' means? Should we state here that this is a circuit whose parameters have been set via the bind/assign so that no symbolic parameters remain (assuming I have it right!). Did it have to be a parameterized circuit in the first place - e.g. is it valid to pass in a circuit with fixed parameters to the transpile the first time around and still use this?

Reply:
I've added more docs in 718bfc3, hopefully it's clearer now 🙂